### PR TITLE
fix(otrs): fix otrs popup style

### PR DESCRIPTION
**Pull request by AxelPeter**
UXMWEB-3905

### CODE REVIEW

File: client/app/components/ovh-angular-otrs/ovh-angular-otrs.less [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
     #headerPopup {
-        position: relative !important;
-        top: 0 !important;
+        position: absolute;
+        top: 0;
     }
 
-    #contentPopup {
-        height: auto !important;
+    #contentPopup { padding: 0 20px 20px 20px; }
```
Comment (antleblanc) [inline client/app/components/ovh-angular-otrs/ovh-angular-otrs.less]: shorthand:

```less
#contentPopup {
    padding: 0 20px 20px;
}
```

File: client/app/components/ovh-angular-otrs/ovh-angular-otrs.less [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+
+    .ovh-form-flat::after {
```
Comment (antleblanc) [inline client/app/components/ovh-angular-otrs/ovh-angular-otrs.less]: Usage of class `.ovh-form-flat` isn't the best path to follow.
First you didn't have this dependency on Web, right?
Secondly this dependency is now deprecated.
@jleveugle Existing another option?

Comment (AxelPeter) [reply client/app/components/ovh-angular-otrs/ovh-angular-otrs.less]: Why ? It's the classname of the form inside `#contentPopup`. I could use `#otrsPopupForm`, but it's just to clear a float. Or put the clear on `#contentPopup`.

File: client/app/components/ovh-angular-otrs/ovh-angular-otrs.less (hunk 1 above)
Comment (AxelPeter) [reply client/app/components/ovh-angular-otrs/ovh-angular-otrs.less]: I prefer to declare all, but it works too

**Comment by jleveugle** [comment]
I will clos this PR because it's not the right place to do this fix.

We should update the otrs module and use the associate stylesheet :)

**Pull request closed without merging**